simtools/io_handler.py

```diff
@@ -59,18 +59,20 @@ def set_paths(
         self.data_path = data_path
         self.model_path = model_path
 
-    def get_output_directory(self, label=None, dir_type=None, test=False):
+    def get_output_directory(self, label=None, sub_dir=None, dir_type="simtools"):
         """
-        Get the output directory for the directory type dir_type
+        Return path to output directory
 
         Parameters
         ----------
         label: str
             Instance label.
-        dir_type: str
+        sub_dir: str
             Name of the subdirectory (ray-tracing, model etc)
-        test: bool
-            If true, return test output location
+        dir_type: str
+            Directory type (e.g., 'simtools', 'test', 'simtools-result')
+            Using 'simtools-result' will return the output directory without
+            subdirectory string added for the use_plain_output_path option.
```

Comment:
I reformulated it. Let me know if this is ok.

Reply:
I am still missing explanations for type. I am not sure whether this would be clear for others but I would not know what to choose as `dir_type`. Is the choice really supposed to be free?

My general feeling is that we don't need a lot of options. Maybe the three examples you gave are sufficient? 

Wouldn't it be better to use the same flag in case of use_plain_output_path, i.e., instead of `simtools-result` we use `use_plain_output_path` for dir_type?